next/src/components/common/NavBar/NavMenu/NavMenuSection.tsx

```diff
@@ -1,41 +1,42 @@
 import { Typography } from '@bratislava/component-library'
 import React from 'react'
 
+import NavMenuLink from '@/src/components/common/NavBar/NavMenu/NavMenuLink'
+import { MenuSection } from '@/src/components/common/NavBar/NavMenu/navMenuTypes'
 import Pictogram from '@/src/components/common/Pictogram/Pictogram'
-import cn from '@/src/utils/cn'
-
-import NavMenuLink from './NavMenuLink'
-import { MenuSection } from './navMenuTypes'
 
 type NavigationSectionProps = {
   section: MenuSection
-  classNames?: string
 }
 
-const NavMenuSection = ({ section, classNames }: NavigationSectionProps) => {
+/**
+ * Figma: https://www.figma.com/design/17wbd0MDQcMW9NbXl6UPs8/DS--Component-library?node-id=19079-20827&m=dev
+ */
+
+const NavMenuSection = ({ section }: NavigationSectionProps) => {
+  const { icon, label, subtext, showMoreLink } = section
+
+  const { href, ...restLinkProps } = showMoreLink ?? {}
+
   return (
-    <li className={cn('flex gap-2', classNames)}>
-      {section.icon && (
-        <div aria-hidden>
-          <Pictogram iconName={section.icon} className="size-10" />
-        </div>
-      )}
-      <div className="w-full">
-        {section.label && (
-          <Typography variant="h5" as="h3" className="mt-1.5">
-            {section.label}
-          </Typography>
+    <li className="px-4 py-3 lg:px-8 lg:py-4">
+      <div className="wrapper-focus-ring relative flex gap-3 focus-within:rounded-sm lg:gap-4">
+        {icon && (
+          <div aria-hidden>
+            <Pictogram iconName={icon} className="size-8 text-category-700 lg:size-10" />
+          </div>
         )}
+        <div className="flex w-full flex-col gap-2">
+          <div className="flex items-center">
+            <NavMenuLink variant="menuSectionLink" href={href ?? '#'} {...restLinkProps}>
+              <Typography variant="h5" as="h3">
+                {label}
+              </Typography>
+            </NavMenuLink>
+          </div>
 
-        <ul className="mt-1.5 flex flex-col">
-          {section.items?.map((menuLink, index) => {
-            // eslint-disable-next-line react/no-array-index-key
-            return <NavMenuLink key={index} {...menuLink} />
-          })}
-          {section.showMoreLink ? (
-            <NavMenuLink variant="showMoreLink" {...section.showMoreLink} />
-          ) : null}
-        </ul>
+          {subtext ? <Typography variant="p-small">{subtext}</Typography> : null}
+        </div>
```

Comment:
I'm a bit confused here.
Before we used `NavMenuLink` to render the final link to a page and we just simple spread `{...menuLink}` onto it.

Now it's wrapped in several divs, `subtext` and `pictogram` are outside of actual link, it uses Typography as children, menuLink props are not just spread, but `href` is handed separately...


Please prepare `href` somewhere above, ideally in getParsedMenu :)
Please Move Typography into `NavMenuLink`.
So `NavMenuLink` can be used as before :) 

Pictogram can stay outside of `NavMenuLink`, because it does not hold any information. But I'm not sure about subtext, if it shouldn't also be part of the link.